ticdc/ticdc-sink-to-kafka.md

```diff
@@ -298,16 +321,16 @@ Starting from v7.3.0, TiCDC Kafka sink supports sending only the handle keys whe
 * If the table to be replicated has primary key, the primary key is the handle key.
 * If the table does not have primary key but has NOT NULL Unique Key, the NOT NULL Unique Key is the handle key.
 
-Currently, this feature supports two encoding protocols: Canal-JSON and Open Protocol. When using the Canal-JSON protocol, you must specify `enable-tidb-extension=true` in `sink-uri`.
-
 The sample configuration is as follows:
 
 ```toml
 [sink.kafka-config.large-message-handle]
-# This configuration is introduced in v7.3.0.
-# Empty by default, which means when the message size exceeds the limit, the changefeed fails.
-# If this configuration is set to "handle-key-only", when the message size exceeds the limit, only the handle key is sent in the data field. If the message size still exceeds the limit, the changefeed fails.
-large-message-handle-option = "handle-key-only"
+# large-message-handle-option is introduced in v7.3.0.
+# Defaults to "none". When the message size exceeds the limit, the changefeed fails.
+# When set to "handle-key-only", if the message size exceeds the limit, only the handle key is sent in the data field. If the message size still exceeds the limit, the changefeed fails.
+# When set to "claim-check", if the message size exceeds the limit, the message is sent to external storage.
```

Comment:
```suggestion
```

Reply:
https://github.com/pingcap/docs-cn/blob/9cc5a36bc3d4ae81797a66372a894cb47ea8b833/ticdc/ticdc-sink-to-kafka.md?plain=1#L327-L335

```toml
[sink.kafka-config.large-message-handle]
# 该参数从 v7.3.0 开始引入
# 默认为空，即消息超过大小限制后，同步任务失败
# 设置为 "handle-key-only" 时，如果消息超过大小，data 字段内容只发送 handle key；如果依旧超过大小，同步任务失败
large-message-handle-option = "handle-key-only"
```